### PR TITLE
Update power usage

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request

This improves the code quality for power and enables power usage for a bunch of items across the board.

## Changelog
:cl:
tweak: power across all the things
/:cl:

### CODE REVIEW

File: code/game/objects/machinery.dm [hunk 1]
```diff
@@ -29,6 +29,7 @@
 	. = ..()
 	GLOB.machines += src
 	component_parts = list()
+	START_PROCESSING(SSmachines, src)
```
Comment (Rohesie) [inline code/game/objects/machinery.dm]: Does every machine need to process on init?

**Comment by Rohesie** [comment]
There are several improvements here. My only concern is for all machines to be added to processing on init. I know /tg/ does this, but it's really not ideal.
Maybe a flag could be added to signify if the machine should be always processing, or the likes.

File: code/game/objects/machinery.dm (hunk 1 above)
Comment (psykzz) [reply code/game/objects/machinery.dm]: If we want a machine to consume power we do.
If a machine doesn't it will be removed on the first tick.

Comment (Rohesie) [reply code/game/objects/machinery.dm]: Now that I think it a bit more, maybe starting either in LateInitialize or after the game mode starts might be the best here. The latter would be ideal IMO, but out of scope. Push it to late init?

Comment (psykzz) [reply code/game/objects/machinery.dm]: Sure

File: code/game/objects/machinery.dm [hunk 2]
```diff
@@ -30,6 +30,11 @@
 	GLOB.machines += src
 	component_parts = list()
```
Comment (Rohesie) [inline code/game/objects/machinery.dm]:
```suggestion
	component_parts = list()
	return INITIALIZE_HINT_LATELOAD
```

**Pull request merged**